traffic_portal/app/src/common/modules/table/servers/table.servers.tpl.html

```diff
@@ -23,31 +23,47 @@
             <li class="active">Servers</li>
         </ol>
         <div class="pull-right">
-            <a class="btn btn-primary" name="createServersButton" title="Create Server" href="#!/servers/new"><i class="fa fa-plus"></i></a>
-            <button class="btn btn-default" title="Refresh" ng-click="refresh()"><i class="fa fa-refresh"></i></button>
-            <div id="toggleColumns" class="btn-group" role="group" title="Select Table Columns" uib-dropdown is-open="columnSettings.isopen">
-                <button type="button" class="btn btn-default dropdown-toggle" uib-dropdown-toggle aria-haspopup="true" aria-expanded="false">
-                    <i class="fa fa-columns"></i>&nbsp;
-                    <span class="caret"></span>
-                </button>
-                <menu ng-click="$event.stopPropagation()" class="column-settings dropdown-menu-right dropdown-menu" uib-dropdown-menu>
-                    <li role="menuitem" ng-repeat="c in gridOptions.columnApi.getAllColumns() | orderBy:'colDef.headerName'">
-                        <div class="checkbox">
-                            <label><input type="checkbox" ng-checked="c.isVisible()" ng-click="toggleVisibility(c.colId)">{{::c.colDef.headerName}}</label>
-                        </div>
-                    </li>
-                </menu>
-            </div>
-            <div class="btn-group" role="group" uib-dropdown is-open="more.isopen">
-                <button type="button" class="btn btn-default dropdown-toggle" uib-dropdown-toggle aria-haspopup="true" aria-expanded="false">
-                    More&nbsp;
-                    <span class="caret"></span>
-                </button>
-                <ul class="dropdown-menu-right dropdown-menu" uib-dropdown-menu>
-                    <li role="menuitem"><a ng-click="confirmCDNQueueServerUpdates(null)">Queue CDN Server Updates</a></li>
-                    <li role="menuitem"><a ng-click="confirmCDNClearServerUpdates(null)">Clear CDN Server Updates</a></li>
-                    <li role="menuitem"><button class="menu-item-button" type="button" ng-click="exportCSV()">Export CSV</button></li>
-                </ul>
+            <div class="form-inline">
+                <div class="input-group text-input">
+                    <span class="input-group-btn">
+                        <button class="btn btn-default" type="button"><i class="fa fa-search"></i></button>
+                    </span>
+                    <input type="text" class="form-control" placeholder="Quick search..." ng-model="quickSearch" ng-change="onQuickSearchChanged()"/>
+                </div>
+                <div class="input-group text-input">
+                    <span class="input-group-btn">
+                        <button class="btn btn-default" type="button">Page Size:</button>
```

Comment:
Why is this a button? Should it be a label?

Reply:
![image](https://user-images.githubusercontent.com/251272/88081956-5c968280-cb3e-11ea-9f0f-313bb4dd1058.png)

just a button for style purposes. both the search icon and the `Page Size` text.

Reply:
i'll fix it